lib/passwordless.rb

```diff
@@ -25,4 +28,12 @@ module Passwordless
 
   CookieDeprecation = ActiveSupport::Deprecation.new("0.9", "passwordless")
   SessionValidDeprecation = ActiveSupport::Deprecation.new("0.9", "passwordless")
+
+  def self.digest(token)
+    key_generator = ActiveSupport::CachingKeyGenerator.new(
+      ActiveSupport::KeyGenerator.new(digest_secret.call)
+    )
+    key = key_generator.generate_key("passwordless")
+    OpenSSL::HMAC.hexdigest(digest_algorithm, key, token)
+  end
```

Comment:
Perhaps it would be nice to move this to its own class. Maybe only create the generator once and save it as an ivar.